src/libsync/theme.cpp

```diff
@@ -370,7 +370,7 @@ QIcon Theme::syncStateIcon(SyncResult::Status status, bool sysTray, bool sysTray
     switch (status) {
     case SyncResult::Undefined:
         // this can happen if no sync connections are configured.
-        statusIcon = QLatin1String("state-information");
+        statusIcon = QLatin1String("state-warning");
```

Comment:
Any praticular reason you changed this?

Reply:
Followed by the discussion in https://github.com/nextcloud/client_theming/issues/196#issuecomment-321990107 

It makes sense to me to unify those, since there is either an error or some state where the user might need to interact, but that is not as critical as the error state.